pkgs/applications/editors/vscode/extensions/squiggle/default.nix

```
@@ -0,0 +1,22 @@
+{ lib
+, vscode-utils
+}:
+let
+  version = "0.5.0";
```

```suggestion
  version = "0.5.0";
```
Is version re-used somewhere? If not, keep in scope.